### ISSUE TITLE
Visual Bug, Bad UI

### ISSUE THREAD

**Issue by evdmx** [The-Poolz/Poolz-Back#40]
**Describe the bug**
The Continue button is very hard to see on some platforms, text is too light. This would fail most accessibility tests for web design.

**To Reproduce**
1. Join any pool

**Expected behavior**
Continue CTA button should be more visible

**Screenshots**
<img width="865" alt="Screen Shot 2021-01-06 at 16 05 02" src="https://user-images.githubusercontent.com/39380100/103790624-5be4a080-5039-11eb-8f1f-13a47865bfe8.png">

**Desktop (please complete the following information):**
- MacOS 10.12.6
- Chrome
- 87.0.4280.88 (Official Build) (x86_64)

**Additional context**
Connected via Metamask on Rinkeby

**Wallet address**
0x1C69ac5509e5fac81e1636f03F9AAa3EB56CdbE6

**Comment by Lomet** [comment]
https://github.com/The-Poolz/Poolz-Back/issues/18
